### PR TITLE
Add sizes for poster thumbnail and pdf

### PR DESCRIPTION
**Pull request by jasongrout**
CC @jwagemann @damianavila

I made the thumbnail size exact instead of preferred, thinking that our gallery page will force the image to be a consistent size anyway.

What would the max poster size be?

### CODE REVIEW

**Comment by jasongrout** [comment]
@jwagemann @damianavila - can you approve (or suggest changes) before we merge?

**Comment by jwagemann** [comment]
Approved. Maybe we can say instead of 'minimum' preferred? In this way, we indicate our preferred size, which is not too small and not too big.

**Comment by jasongrout** [comment]
> Maybe we can say instead of 'minimum' preferred? In this way, we indicate our preferred size, which is not too small and not too big.

At first I was going to suggest A1 be the preferred size, but is the reasoning that online posters don't have to have such large font, and scrolling is hard on a computer screen anyway, so you can fit much more content in an A2 size? Perhaps we can suggest A2 or 16x24 inches to make it easier for US folks?

**Comment by jwagemann** [comment]
Yes, I'd say so - A2 is the preferable size.

File: pages/talk-poster-cfp.vue [hunk 1]
```diff
@@ -85,8 +85,8 @@
                 </p>
 
                 <ul>
-                    <li> A small thumbnail image (JPG or PNG) for our poster gallery</li>
-                    <li> A poster pdf file</li>
+                    <li> A small thumbnail image (JPG or PNG) for our poster gallery (1280x720 pixels)</li>
+                    <li> A poster pdf file (minimum size: A2 or 16.5 x 23.4 inches)</li>
```
Comment (jasongrout) [inline pages/talk-poster-cfp.vue:89]:
```suggestion
                    <li> A small thumbnail image (JPG or PNG) for our poster gallery (1280x720 pixels)</li>
                    <li> A poster pdf file (preferred size: A2 or 16.5 x 23.4 inches)</li>
```

File: pages/talk-poster-cfp.vue [hunk 2]
```diff
@@ -85,8 +85,8 @@
                 </p>
 
                 <ul>
-                    <li> A small thumbnail image (JPG or PNG) for our poster gallery</li>
-                    <li> A poster pdf file</li>
+                    <li> A small thumbnail image (JPG or PNG) for our poster gallery (1280x720 pixels)</li>
+                    <li> A poster pdf file (preferred size: A2 or 16.5 x 23.4 inches)</li>
```
Comment (jasongrout) [inline pages/talk-poster-cfp.vue:89]:
```suggestion
                    <li> A poster pdf file (preferred size: A2, 16.5 x 23.4 inches)</li>
```

**Comment by jasongrout** [comment]
Great, merged! Thanks again!

@SylvainCorlay @martinRenou - can you deploy?

**Comment by jtpio** [comment]
> can you deploy?

Auto-deploys should be back now.

**Pull request merged**